packages/core/src/connectivity/scp-cf/destination/destination-from-service.ts

```diff
@@ -134,6 +134,7 @@ class DestinationFromServiceRetriever {
   private static async getProviderClientCredentialsToken(
     options: DestinationOptions
   ): Promise<string> {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Instead of removing the linter you could try: 
`const optionsWithoutJwt:Omit<DestinationOptions,'userJwt'> = options; `

Reply:
Problem is not the type but `userJwt` which is an unused variable that is needed to use this short snippet to copy an object with that one property removed. I am not aware of any shorter version to copy+remove property so I just kept it as is.